src/performance/connection.ts

```diff
@@ -0,0 +1,31 @@
+import { BooleanFlag, Event } from "@clarity-types/data";
+import { ConnectionData, NavigatorConnection } from "@clarity-types/performance";
+import encode from "./encode";
+
+export let data: ConnectionData;
+
+export function start(): void {
+    if (navigator && "connection" in navigator) {
+        (navigator["connection"] as NavigatorConnection).addEventListener("change", recompute);
+        recompute();
+    }
+}
+
+function recompute(): void {
+    let connection = navigator["connection"] as NavigatorConnection;
```

Comment:
>NavigatorConnection [](start = 48, length = 19)

Add comment with link to [http://wicg.github.io/netinfo/#dom-networkinformation](http://wicg.github.io/netinfo/#dom-networkinformation)

Reply:
That's already in performance.d.ts. I had the link and the interface.